app/markdown/2020-10-05/urllib3-newsletter-september-2020.md

```diff
@@ -0,0 +1,144 @@
+# urllib3 Newsletter - September 2020
+
+Welcome to the first urllib3 newsletter! I wanted to put together
+this newsletter to broadcast all the work that the urllib3
+team has done the past month and to share a few very exciting
+announcements!
+
+## Open Collective and GitCoin Grants
+
+urllib3 created an [Open Collective](https://opencollective.com/urllib3)
+which we are now accepting donations and offering sponsorship to organizations and individuals.
+Primarily we are looking for support to pay for core contributors to
+work on urllib3 `v2.0.0`.
+
+If cryptocurrency is more your style, we also have an
+[ongoing grant through GitCoin Grants](https://gitcoin.co/grants/65/urllib3)
+that allows you to support Open Source with many popular cryptocurrencies.
+
+Thanks for considering supporting our team!
+
+## v2.0 Roadmap and Features
+
+The most exciting item is we released our [v2.0 Roadmap in the documentation](https://urllib3.readthedocs.io/en/latest/v2-roadmap.html)
+and the [v2.0 Milestone on GitHub](https://github.com/urllib3/urllib3/milestone/6). 
+
+You can read a whole lot more about what will be in the `v2.0.0` release
+there. Here are some of the highlights:
+
+– Drop support for Python 2.7 and 3.5
+– 99% Functional API Compatibility with urllib3 `v1.x`
+– Fully Type-Hinted APIs
+– Use TLS 1.2+ by default with HTTPS
+– Stop verifying deprecated `commonName` fields in Certificates
+– Tracing information on the `HTTPResponse` object
+
+There will be more updated on `v2.0.0` development after we ship the
+release of urllib3 `v1.26.0`.
+
+## HTTPS in HTTPS Tunnel Proxies
+
+The major new feature coming in urllib3 `v1.26.0` is being able to
+use HTTPS proxies to tunnel to HTTPS hosts.
+
+Previously this wasn't possible, you would be forced to use HTTP to
+connect to your proxy and then create an HTTPS tunnel to the origin
+through your proxy. This means that the connection between you and
+your proxy wasn't secure and only the connection to the origin was secure.
+
+*Obviously not ideal!* Combine this with HTTPS proxies being more
+common and there's an obvious demand for this functionality.
```

Comment:
One use case we can mention is that this allows the proxy to authenticate clients, which is an useful way to replace passwords in a corporate settings. I think this was part of @jalopezsilva's motivation here.